control-plane/pkg/reconciler/broker/controller.go

```diff
@@ -119,6 +119,10 @@ func NewController(ctx context.Context, watcher configmap.Watcher, env *config.E
 	reconciler.Tracker = impl.Tracker
 
 	secretinformer.Get(ctx).Informer().AddEventHandler(controller.HandleAll(reconciler.Tracker.OnChanged))
+	secretinformer.Get(ctx).Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: controller.FilterWithName(brokerIngressTLSSecretName),
+		Handler:    controller.HandleAll(globalResync),
+	})
```

Comment:
Since we are tracking all changes in secrets in the line above already, we probably don't have to specifically listen for changes to `brokerIngressTLSSecretName`. 

Can you check whether the tests pass without adding this line